lib/dor/was_seed/structural_builder.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Dor
+  module WasSeed
+    # Builds a Cocina representation of the structural metadata
+    class StructuralBuilder
+      # Generates structural metadata for a repository object.
+      #
+      # @param [Cocina::Models::DRO] cocina_model
+      # @param [Array<Array<Assembly::ObjectFile>>] objects the list of files to add to structural metadata
```

Comment:
Isn't there always only a single ObjectFile? Seems like this has additional complexity to support an array.